src/components/trace/ha-trace-path-details.ts

```diff
@@ -212,7 +212,7 @@ export class HaTracePathDetails extends LitElement {
       <div class="padded-box">
         ${data.map(
           (trace, idx) => html`
-            ${idx > 0
+            ${data.length > 1
```

Comment:
If `data.length === 0` we would never get here right? So this check seems useless?

Reply:
Sure, if `data.length === 0` we don't get here. But if `data.length === 1` we sure do, and that's the case fixed by this PR.

Reply:
Nevermind I reviewed way too hurried 😅